graylog2-web-interface/src/views/components/aggregationbuilder/EventListConfiguration.jsx

```diff
@@ -0,0 +1,34 @@
+// @flow strict
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { Checkbox, FormGroup } from 'components/graylog';
+
+type Props = {
+  enabled: boolean,
+  onChange: (value: boolean) => void,
+};
+
+const EventListConfiguration = ({ enabled, onChange }: Props) => {
+  return (
+    <form>
+      <FormGroup>
+        {/* eslint-disable-next-line no-undef */ /* $FlowFixMe: checked is part of target */}
+        <Checkbox onChange={(event: SyntheticEvent<HTMLInputElement>) => onChange(event.target.checked)} checked={enabled}>
+          Enable Event Annotation
+        </Checkbox>
+      </FormGroup>
+    </form>
+  );
+};
+
+EventListConfiguration.propTypes = {
+  enabled: PropTypes.bool,
+  onChange: PropTypes.func,
+};
```

Comment:
Mainly because we are describing these attributes as required in the flow type definition, I would do the same for the PropType definition.

Reply:
Since both values have defaults, they should never be optional for flow.